drop-support-system-packages.rst

```diff
@@ -0,0 +1,46 @@
+.. post:: August 8, 2023
+   :tags: builders
+   :author: Manuel
+   :location: BCN
+   :category: Changelog
+
+Drop support for "Use system packages"
+======================================
+
+You are receving this email because your Read the Docs project is impacted by an upcoming deprecation.
```

Comment:
This isn't true, right?

Reply:
```suggestion
```

Reply:
He, you are right. I should have deleted this paragraph after copying and pasting it from the hackmd.io email template 😄